ARCs/arc-0031.md

```diff
@@ -0,0 +1,269 @@
+---
+arc: 31
+title: Authentication with Algorand accounts
+description: Use Algorand accounts to authenticate with third-party services
+author: Stefano De Angelis (@deanstef)
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/42
+status: Draft
+type: Meta
+created: 2022-04-05
+---
+
+# Authentication with Algorand accounts
+
+A standard for authentication with Algorand accounts.
+
+## Abstract
+
+This ARC introduces a standard for authenticating users with their Algorand accounts. It leverages asymmetric encryption <*PK, SK*> to verify the identity of a user, owner of an Algorand account. This approach fosters the adoption of novel identity management systems for both Web3 and Web2 applications.
+
+### Definitions
+
+- **System**: any frontend/backend application, service provider, or in general an entity not based on blockchain;
+- **Credentials**: any type of authentication used by users to access their online accounts, e.g. username/password, PIN, public/secret key pair;
+- **Blockchain identity**: a public/secret key pair <*PK, SK*> representing a blockchain account;
+- **Algorand account**: a blockchain identity on Algorand identified with the key pair <*PKa, SKa*>;
+- **Algorand address**: the public key *PKa* of an Algorand account;
```

Comment:
Technically the address is not exactly the public key as there is additional base32 encoding + checksum. I see the distinction is made clear later.